src/jailer/src/main.rs

```diff
@@ -266,6 +270,11 @@ pub fn build_arg_parser() -> ArgParser<'static> {
                 .takes_value(true)
                 .help("Arguments that will be passed verbatim to the exec file."),
         )
+        .arg(Argument::new("cgroup").allow_duplicates(true).help(
```

Comment:
That was my first approach (using multiple instead of duplicates) but as there is an specific error for "duplicates arguments" I used the same name. For me it would be Ok to renaming to multiple if you think it would be easier to understand.

Regarding having both as in your example I think that would be the best approach if we decide to check for duplicates cgroups.

Reply:
I've said before, as a reply to previous comments, that we can use the policy of _persist the last cgroup definition_. This means that we do not need to check for duplicate cgroups anymore. However, the naming here might fail us in understanding the difference between duplicate and multiple, so I would also suggest to differentiate that in case of `--cgroup` we have `multiple` arg rather than `duplicate` arg. What do you think?

Reply:
Ok, I think it's reasonable. I'll change it.